api4/oauth_test.go

```diff
@@ -35,14 +35,8 @@ func TestCreateOAuthApp(t *testing.T) {
 	rapp, resp := AdminClient.CreateOAuthApp(oapp)
 	CheckNoError(t, resp)
 	CheckCreatedStatus(t, resp)
-
-	if rapp.Name != oapp.Name {
-		t.Fatal("names did not match")
-	}
-
-	if rapp.IsTrusted != oapp.IsTrusted {
-		t.Fatal("trusted did no match")
-	}
+	require.Equal(t, rapp.Name, oapp.Name, "names did not match")
+	require.Equal(t, rapp.IsTrusted, oapp.IsTrusted, "trusted did no match")
```

Comment:
use first expected value (oapp.x in this case) first and then results (rapp.x) for consistency, as this is the expected behaviour on testify